lib/mongoose_push/service/fcm/pools.ex

```diff
@@ -18,27 +18,17 @@ defmodule MongoosePush.Pools do
   def worker_name(type, name, num), do: String.to_atom(~s"#{type}_#{name}_#{num}")
 
   @doc "Returns lists of pool names that have selected `:mode` set"
-  @spec pools_by_mode(MongoosePush.service(), MongoosePush.mode()) :: list(atom)
-  def pools_by_mode(:fcm = service, _mode) do
-    config = pools_config(service)
+  @spec pools_by_mode() :: list(atom)
+  def pools_by_mode() do
+    config = pools_config(:fcm)
```

Comment:
Let's remove `import MongoosePush.Application` and use `alias MongoosePush.Application` instead for improved readability.